govcd/vm.go

```diff
@@ -554,6 +554,43 @@ func (vm *VM) HandleInsertMedia(org *Org, catalogName, mediaName string) (Task,
 	})
 }
 
+// Helper function which finds media, calls EjectMedia, waits for task to complete and answer question.
+// Also waits until vm status refreshes - this added as current VCD version has lag in status update.
```

Comment:
1.) Description of what `answerYes` does needed.

2.) Proper capitalisation needed:
"vm" -> "VM"
"VCD" -> "vCD"

Reply:
fixed